### DISCUSSION TITLE
[FYI] Docker image from Linuxserver.io available!

### DISCUSSION THREAD

**Discussion by ghost** [pyload/pyload#3975] [Announcements]
Just sharing a quick announcement that LinuxServer.io has just published (on my humble suggestion) a Docker image for Pyload: https://hub.docker.com/r/linuxserver/pyload

LinuxServer images are very well-supported high-quality images (lightweight, frequent base image rebase, running as non-root...) so I would definitely recommend anyone running pyload in Docker to have a look at it.

X-ref #3398

**Comment by vuolter** [comment]
Looks promising! 😃

**Comment by GammaC0de** [comment]
Thank you, nice job!

pinned.

**Comment by kmmao** [comment]
default login is: username - admin password - password？

**Comment by GammaC0de** [comment]
@kmmao 
> default login is: username - admin password - password？


Yes, See https://github.com/linuxserver/docker-pyload#application-setup

**Comment by devster31** [comment]
Does this mean that the `pyload/pyload` image mentioned in the README (on master branch) won't be published or is that a future plan?

**Comment by GammaC0de** [comment]
pyLoad uses the `stable` branch, `master` branch is future version, not ready yet.
